dev-docs/modules/userid-submodules/liveintent.md

```diff
@@ -148,6 +209,12 @@ NOTE: For optimal performance, the LiveIntent ID sub-module should be called at
 | params.liCollectConfig.fpiExpirationDays |Optional| Number |The expiration time of an identifier created and updated by LiveConnect. By default, 730 days.|`729`|
 | params.liCollectConfig.collectorUrl |Optional| String |The parameter defines where the signal pixels are pointing to. The params and paths will be defined subsequently. If the parameter is not set, LiveConnect will by default emit the signal towards `https://rp.liadm.com`.| `https://rp.liadm.com`|
 | params.liCollectConfig.appId |Optional| String |LiveIntent's media business entity application ID.|`a-0012`|
+| params.sharedId.name | Optional | String | The parameter is cookie/localstorage key name | `'__super_duper_cookie'`|
+| params.sharedId.strategy | Optional | String | The parameter defines where to get the identifier from. Either from the cookie jar, `'cookie'`, or from the local storage, `'localstorage'`. | `'localstorage'`|
```

Comment:
Below there is an example "If all the supported configuration params are passed, then the setup will look as follows:".  This should be added to that example.